.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/vscode/devcontainers/java:0-16
+
+# [Option] Install Maven
+ARG INSTALL_MAVEN="false"
```

Comment:
I was very confused to see Maven AND Gradle, but then I realized that these were options that you abandoned and/or were demonstrating could also be used